webpack/sass/main.scss

```diff
@@ -211,8 +217,18 @@ code {
 }
 
 .single-post {
-  .container {
-    @extend .padding-horizontal-xxl;
+  @media only screen and (min-width: 768px) {
```

Comment:
you should take advantage of the [responsive functions](https://bulma.io/documentation/overview/responsiveness/) from bulma
you can use them like:
```
@include mobile() {
  ...
}

Reply:
On it.